src/store/Chat/index.js

```diff
@@ -10,6 +10,7 @@ const initialState = {
   activeIsOpen: null,
   chats: [], // DUMMY_DATA ? dummy.chatSessions : [],
   messages: [], // DUMMY_DATA ? dummy.messages : [],
+  typing: {},
```

Comment:
I ultimately decided to keep typing state separate from chats. It's just a lot easier to read and update.